cognite/neat/core/_data_model/_shared.py

```diff
@@ -32,7 +32,7 @@ class ImportedDataModel(Generic[T_UnverifiedDataModel]):
     """
 
     unverified_data_model: T_UnverifiedDataModel | None
-    context: dict[str, SpreadsheetRead]
+    context: Mapping[str, object]
```

Comment:
Making this a mapping instead of dict to signal that this is intended to be Immutable when you consume it.